sources/data/itemDataActivity.js

```diff
@@ -0,0 +1,26 @@
+class ItemDataActivity {
```

Comment:
This class is identical to ItemDataContact. Create one class and export two different object from two different files